examples/powertools-examples-core/pom.xml

```diff
@@ -118,4 +122,72 @@
         </plugin>
       </plugins>
     </build>
+    <profiles>
```

Comment:
Add a profile to override the aspectj setup for Java 1.8. All other java versions build happily with the default configuration.